switch-configuration/config/scripts/template.pl

```diff
@@ -1,15 +1,139 @@
 #!/usr/bin/perl
```

Comment:
What do you think about `#!/usr/bin/env perl` so it compatible with the BSD folks?

Reply:
I think that would break Mac and most Linux installations.

Yes, the named references are intended. That's the usual Juniper way. The script doesn't yet write out the "vlans" section of the configuration file (I'm actively working on that code as I write this), which provides the association between numbers and names. (It also does the definitions necessary for private vlans, including community and isolated secondary private vlans.

Reply:
@owendelong I dont think that would break anything. I can double check on my mac tomorrow if you want. I usually use `#!/usr/bin/env <interpreter>` for most things I write.

Thanks for the clarification about the the refs.

Reply:
OK... Tested it and it does appear to work on Mac and Fedora. Will make the change.

Reply:
hehe yay, thanks! My FreeBSD laptop appreciates it!